rrule/src/parser/error.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(clippy::module_name_repetitions)]
 use thiserror::Error;
 
 #[derive(Error, Debug, Clone, PartialEq)]
```

Comment:
I cannot comment down there, and it's not your changes, but around line 64, there is a typo, parameter